controller/controller_test.go

```diff
@@ -1117,6 +1117,30 @@ func TestController_UpdateValidation(t *testing.T) {
 				ReadySecs: 1,
 			},
 		},
+		// Test that growth of 8, a minalive of 3, with one running unit, does return an error.
```

Comment:
minalive is set to 1 in the request in the test.
I guess its not important for the test though.